news/11114.bugfix.rst

```diff
@@ -0,0 +1 @@
+Pass ``--disable-pip-version-check`` when running pip in tests to fix test failures due to upgrade check warnings.
```

Comment:
This is not a user-facing bug fix. This should be renamed as trivial.

Reply:
Done.